HIP/hip-752.md

```diff
@@ -0,0 +1,482 @@
+---
+hip: 752
+**title: Partitioning Functionality for Fungible Tokens on Hedera**
+author: Sam Wood <sam.wood@toko.network>, Stephanie Yi <stephanie.yi@toko.network>, Eleonora Odorizzi <eleonora.odorizzi@toko.network>, Vicky Lio <vicky.lio@toko.network>
```

Comment:
It is encouraged that a hip have one champion and the other contributors be listed in a working-group like this:

```suggestion
author: Sam Wood <sam.wood@toko.network>
working-group: Stephanie Yi <stephanie.yi@toko.network>, Eleonora Odorizzi <eleonora.odorizzi@toko.network>, Vicky Lio <vicky.lio@toko.network>
```